### PR TITLE
fix(color): Clear Lua lcd buffer and display 'Loading...'

### LINKED ISSUE #2810: Screen buffer corruption on standalone Lua launch
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

Synchronization failed while creating the model -wizard - plane. 
Appears on Edge TX 2.7.1 and 2.8

https://www.youtube.com/watch?v=_PE6sHLRN58

### Expected Behavior

Appears on Edge TX 2.7.1 and 2.8
TX16S. Bought recently from Amazon at the Radiomaster store.
Also, the error is detected in many people who bought the transmitter 1.5 years ago.
Most likely a software error.

### Steps To Reproduce

https://www.youtube.com/watch?v=_PE6sHLRN58

### Version

2.8.0

### Transmitter

Radiomaster TX16S / TX16SMK2

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by philmoz**
Fixes #2810

Summary of changes:

Clear the Lcd buffer and display "Loading..." text in the constructor for StandaloneLuaWindow

### CODE REVIEW

**Comment by ajjjjjjjj** [comment]
PL
`#define TR_LOADING "Ładowanie..."`

**Comment by pfeerick** [comment]
Can we get some translations for this please 😃 

It gets written to the middle of the screen while a Tools Lua script is being loaded

```
#define TR_LOADING "Loading..."
```

@HThuren DA
@ParkerEde DE
@Eldenroot CZ
@ulfhedlund SE
@robustini IT
@zyren CN / TW
~~@ajjjjjjjj PL~~ Thanks for beating me to the punch :)
@ToshihiroMakuuchi JP

**Comment by TheIsotopes** [comment]
DE is already in the review, but here again
`#define TR_LOADING                     "Wird geladen..."`

**Review by TheIsotopes — APPROVED** [review]

File: radio/src/translations/de.h [hunk 1]
```diff
@@ -760,8 +760,8 @@
 #define TR_BAUDRATE                    "Baudrate"
 #define TR_MAXBAUDRATE                 "Max Baud"
 #define TR_SAMPLE_MODE                 "Abtastmodus"
-
 #define TR_SAMPLE_MODES                "Normal","OneBit"
+#define TR_LOADING                     "Loading..."
```
[the remark this answers is not in the document]
Comment (pfeerick) [reply radio/src/translations/de.h]: Did you push the save review late or something? As this is already been translated ;) 

image

Something seems out of sync... you mentioned in a comment 38 minutes ago it was already in the review, and then the review is from 11 minutes ago... 
image

**Comment by Eldenroot** [comment]
CZ: Načítání...

File: radio/src/translations/de.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheIsotopes) [reply radio/src/translations/de.h]: imo everything is fine

**Comment by robustini** [comment]
IT
#define TR_LOADING "Caricamento..."

**Comment by ulfhedlund** [comment]
SE#define TR_LOADING "Laddar..."Den 22 dec. 2022 10:19 skrev Peter Feerick ***@***.***>:

Can we get some translations for this please 😃

It gets written to the middle of the screen while a Tools Lua script is being loaded

#define TR_LOADING "Loading..."



@HThuren DA

@ParkerEde DE

@Eldenroot CZ

@ulfhedlund SE

@robustini IT

@zyren CN / TW

@ajjjjjjjj PL Thanks for beating me to the punch :)

@ToshihiroMakuuchi JP

**Comment by ToshihiroMakuuchi** [comment]
JP
#define TR_LOADING "読み込み中..."

**Comment by HThuren** [comment]
DK
#define TR_LOADING "Indlæser..."

**Comment by zyren** [comment]
***************************************
CN
#define TR_LOADING "加载中..."
***************************************
***************************************
TW
#define TR_LOADING "加載中..."
***************************************

**Pull request merged**